NodeBase/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN echo "${TZ}" > /etc/timezone \
 RUN apt-get update -qqy \
   && apt-get -qqy install \
     xvfb \
+    x11-xserver-utils \
```

Comment:
This package provides `xset`. It's already on the Chrome images as a dependency, this just adds it to the other ones.